modules/icon-theme/hm.nix

```
@@ -0,0 +1,121 @@
+{ pkgs, lib, config, ... }:
+
+let
+  cfg = config.stylix.targets.iconTheme;
+in
+{
+  options.stylix.targets.iconTheme = {
+    enable = config.lib.stylix.mkEnableTarget "iconTheme" true;
```

This string is used for the docs so it should be something human readable.

> Whether to style «name».

```suggestion
    enable = config.lib.stylix.mkEnableTarget "the icon theme" true;
```